manual/deploy/deploy_with_nginx.md

```diff
@@ -141,28 +160,16 @@ The following options must be modified in the CONF file:
 
 Optional customizable options in the seafile.conf are:
 
-* Server listening port (listen)- if Seafile server should be available on a non-standard port
+* Server listening port (listen) - if Seafile server should be available on a non-standard port
 * Proxy pass for location / - if Seahub is configured to start on a different port than 8000
 * Proxy pass for location /seafhttp - if seaf-server is configured to start on a different port than 8082
 * Maximum allowed size of the client request body (client_max_body_size)
 
-The default value for `client_max_body_size` is 1M. Uploading larger files bigger will result in an error message HTTP error code 413 ("Request Entity Too Large"). It is recommended to syncronize the value of client_max_body_size with the parameter `max_upload_size` in section `[fileserver]` of [seafile.conf](../config/seafile-conf.md). Optionally, the value can also be set to 0 to disable this feature.
-
-Client uploads are only partly effected by this limit. With a limit of 100 MiB they can safely upload files of any size.
+The default value for `client_max_body_size` is 1M. Uploading larger files will result in an error message HTTP error code 413 ("Request Entity Too Large"). It is recommended to syncronize the value of client_max_body_size with the parameter `max_upload_size` in section `[fileserver]` of [seafile.conf](../config/seafile-conf.md). Optionally, the value can also be set to 0 to disable this feature. Client uploads are only partly effected by this limit. With a limit of 100 MiB they can safely upload files of any size.
 
-Note for very large files (> 4GB): By default Nginx will buffer large request bodies in temp files. After the body is completely received, Nginx will send the body to the upstream server (seaf-server in our case). But it seems when the file size is very large, the buffering mechanism dosen't work well. It may stop proxying the body in the middle. So if you want to support file uploads larger than 4GB, we suggest to install Nginx version >= 1.8.0 and add the following options to Nginx config file:
```

Comment:
Is this note no longer needed?

Reply:
I don't think it is needed. Three reasons:
1.) Nginx 1.8 is from 2015. If a admin still uses Nginx < 1.8.0, problems when uploading large files to Seafile is probably only a very small problem of his.
2.) Over the last three years, we have not used the parameter proxy_request_buffering off;  (while using recent versions of Nginx). We are not aware of any problems. We suspect that the problem was fixed.
3.) This paragraph was redundant: In "Deploy with nginx" and in "Enabling SSL"

Reply:
You have to use GIT command line to add and upload files, not convenient. I have added them.